make/doc.mk

```diff
@@ -60,7 +60,7 @@ HTML_DEPS?=$(wildcard $(APP_EBIN_DIR)/*.beam) $(wildcard *.md) $(wildcard */*.md
 
 $(HTMLDIR)/index.html: $(HTML_DEPS) docs.exs
 	$(gen_verbose)ERL_FLAGS="-pz $(ERL_TOP)/erts/ebin" \
-	  $(EX_DOC) $(EX_DOC_FORMATS) --homepage-url "$(INDEX_DIR)/index.html" "$(APPLICATION)" $(VSN) $(APP_EBIN_DIR) -o "$(HTMLDIR)" -c $(ERL_TOP)/make/ex_doc.exs
+	  $(ESCRIPT) $(EX_DOC) $(EX_DOC_FORMATS) --homepage-url "$(INDEX_DIR)/index.html" "$(APPLICATION)" $(VSN) $(APP_EBIN_DIR) -o "$(HTMLDIR)" -c $(ERL_TOP)/make/ex_doc.exs
```

Comment:
1. If you don't use a specific path to `ex_doc`, this fails because `EX_DOC` isn't a fully resolved PATH apparently
2. If `EX_DOC` ends up being the `ex_doc_wrapper` it will fail because `escript` can't execute a BASH script